src/constants/markets.ts

```diff
@@ -39,6 +42,7 @@ export enum MarketSorting {
 
 export enum MarketFilters {
   ALL = 'all',
+  FAVORITED = 'favorited',
```

Comment:
nit: inconsistent use of favorited vs favorite - i vote just stick with favorite